source/features/repo-age.tsx

```diff
@@ -4,50 +4,108 @@ import cache from 'webext-storage-cache';
 import select from 'select-dom';
 import repoIcon from 'octicon/repo.svg';
 import elementReady from 'element-ready';
+import fetchDom from '../libs/fetch-dom';
 import features from '../libs/features';
-import * as api from '../libs/api';
-import {getRepoGQL, getRepoURL} from '../libs/utils';
+import {getCleanPathname, getRepoURL} from '../libs/utils';
 
 const dateFormatter = new Intl.DateTimeFormat('en-US', {
 	year: 'numeric',
 	month: 'long',
 	day: 'numeric'
 });
 
-const getRepoCreationDate = cache.function(async (): Promise<string> => {
-	const {repository} = await api.v4(`
-		repository(${getRepoGQL()}) {
-			createdAt
-		}
-	`);
+type OldestCommitDetails = {
+	url: string;
+	datetime: string;
+};
 
-	return repository.createdAt;
+const getOldestCommitDetails = cache.function(async (): Promise<OldestCommitDetails | void> => {
+	await elementReady('.commit-tease-sha');
+	const commitsCount = Number(select('li.commits .num')!.textContent!.replace(',', ''));
+	const lastCommitHash = select<HTMLAnchorElement>('.commit-tease-sha')!.href.split('/').pop();
+
+	if (commitsCount === 0) {
+		return;
+	}
+
+	const oldestCommit = await fetchDom(
+		`${getCleanPathname()}/commits?after=${lastCommitHash!}+${commitsCount - 2}`, 'ol:last-child > li.commits-list-item'
+	);
+
+	return {
+		url: select<HTMLAnchorElement>('.commit-title [href*="/commit/"]', oldestCommit)!.href,
+		datetime: select('relative-time', oldestCommit)!.getAttribute('datetime')!
+	};
 }, {
 	cacheKey: () => __featureName__ + ':' + getRepoURL()
 });
 
+const addRepoAgeItem = (element: Element): void => {
+	const license = select('.numbers-summary .octicon-law');
+
+	if (license) {
+		license.closest('li')!.before(element);
+	} else {
+		select('.numbers-summary')!.append(element);
+	}
+};
+
 async function init(): Promise<void> {
-	const date = new Date(await getRepoCreationDate());
+	const placeholder = (
+		<li className="text-gray">
+			{repoIcon()} <span className="text-emphasized js_repo_age_placeholder">Loading...</span>
```

Comment:
No snake case
Refined GitHub classes start with `rgh-`
Probably you don't need a class at all; just select `span`